airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1166,7 +1166,7 @@ paths:
         "404":
```

Comment:
should we do the same thing we did for the 422s to the 404s as well? can't remember if we said were going to hold off for some reason.

Reply:
That's a good idea.  The bit I was holding off on was trying to figure out if "id" ought to be its own attribute; I think it can be.  Every scenario has one.  I will include this change as well.